composer.json

```diff
@@ -25,6 +25,9 @@
 		"data-values/interfaces": "~0.2.0|~0.1.5",
 		"data-values/common": "~0.3.0|~0.2.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8"
```

Comment:
Yes, if you happened to have the correct version of phpunit installed. If you didn't, you had to guess which version would work.